misc/make-deps.sh

```diff
@@ -2,6 +2,8 @@
 # setup a simple go environment
 XPWD=`pwd`
 ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && cd .. && pwd )"	# dir!
+VAGRANT_MODE=
```

Comment:
You should have this ="" for example. So bash with unset var warnings won't happen IIRC.